L1Trigger/L1THGCal/src/HGCalTriggerTools.cc

```diff
@@ -53,6 +58,8 @@ GlobalPoint HGCalTriggerTools::getTCPosition(const DetId& id) const {
     // to distinguish a TC detId
   }
 
+  // FIXME: to do HFNose
```

Comment:
should this be fixed in this PR?

Reply:
In principle it is ok, since it delegates the calculation to the geometry which correctly computes the nose TC positions.
comment can be removed